docs/project_defn.rst

```diff
@@ -252,6 +252,12 @@ on the `GCP Batch <https://cloud.google.com/batch>`_ service.
     * ``repository``: The name of the GCP Artifact Repository in which Docker images are stored.
       This will be combined with the ``project`` and ``region`` to build the full URL to the
       repository.
+*  ``job_environment``: Optional. Specifies the computing requirements for each simulation.
+
+    * ``vcpus``: Number of CPUs needed. Default: 1.
+    * ``memory_mib``: Amount of RAM memory needed for each simulation in MiB. Default: 1024. For large multifamily buildings
+      this works better if set to 2048.
+    * ``machine_type``: GCP Compute Engine machine type to use. If omitted, GCP batch will choose a machine type based on the requested vCPUs and memory.
```

Comment:
what's the relationship between machine_type and vcpus? A machine_type has a fixed number of vcpus (right?), so what does specifying vcpus do when a machine_type is specified?

Reply:
vCPUs and memory are both the amount allocated for each task, and a single machine will handle multiple tasks in parallel if it has enough resources. I'll update these docs to be clearer about that.